lib/stronger_parameters/version.rb

```diff
@@ -1,4 +1,4 @@
 # frozen_string_literal: true
 module StrongerParameters
-  VERSION = '2.21.1'
+  VERSION = '2.22.0'
```

Comment:
I wasn't sure if I should be the one doing the bump or I should focus on the feature and let the maintainers bump.

Reply:
usually no, but does not matter much